src/ftdc_tools/rollups/client_perf.py

```diff
@@ -58,13 +78,21 @@ def add_doc(self, doc: FTDCDoc) -> None:
         else:
             duration = float(doc["timers"]["duration"])
         extracted_duration = duration - self.previous_duration
-
+        number_of_ops = doc["counters"]["ops"] - self.previous_ops
+        number_of_ops = (
+            1 if number_of_ops == 0 else number_of_ops
+        )  # For multi-operation events that have same ops
+        self._operations_total = self._operations_total + number_of_ops
```

Comment:
Do we need to add to this sum every time? The ops are cumulative so the line deleted further down of `self._operations_total = self.last_doc["counters"]["ops"]` should still work even for multi-operation events, so we can still use that at the end and not have `n` executions of this line

Reply:
This is to handle cases like below where all ops are run in parallel and they are all 0.

![image](https://user-images.githubusercontent.com/78040805/197282068-5e239e6d-d604-439c-aa01-f64f4a90d8af.png)

Reply:
Got it. Will fix it